staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -227,7 +227,7 @@ func startCompactorOnce(c storagebackend.TransportConfig, interval time.Duration
 	}, nil
 }
 
-func newETCD3Storage(c storagebackend.Config, newFunc func() runtime.Object) (storage.Interface, DestroyFunc, error) {
+func newETCD3Storage(c storagebackend.Config, newFunc func() runtime.Object, objectCountCallback func(string, int64)) (storage.Interface, DestroyFunc, error) {
```

Comment:
You shouldn't need to pass objectCountCallback here - you already added it to storagebackend.Config, right?

Reply:
yes, fixed it, i overlooked it.